exe_1.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+def findPermutations(str1, str2):
+	str1 = str1.lower()
+	str2 = str2.lower()
+
+	letters_and_frequencies_1 = make_letters_and_frequencies(str1)
+	letters_and_frequencies_2 = make_letters_and_frequencies(str2)
+	return compare_dicts(letters_and_frequencies_1, letters_and_frequencies_2)
+
+def make_letters_and_frequencies(string):
+	lf = {}
+	for char in string:
+		if char in lf:
+			lf[char] += 1
+		else:
+			lf[char] = 1
+	return lf
+
+def compare_dicts(dict1, dict2):
```

Comment:
Good :) That fixes the correctness. Do you think there is a way to solve this without comparing the dicts twice. (This is just to exercise your problem solving, you don't necessarily you have to implement it).